pkg/logql/log/pattern/lexer.rl

```diff
@@ -13,11 +13,25 @@ package pattern
     }
 }%%
 
+%%{
+utf8 = (
```

Comment:
this bit was generated by copilot but I've manually checked it and I think its correct. As far as I understand, anyway.